domain/notifier.go

```diff
@@ -9,6 +9,7 @@ type NotificationMessages struct {
 	OthersAppealApproved string `mapstructure:"others_appeal_approved"`
 	GrantOwnerChanged    string `mapstructure:"grant_owner_changed"`
 	UnusedGrant          string `mapstructure:"unused_grant"`
+	NewComment           string `mapstructure:"new_comment"`
```

Comment:
Shouldn't these be under a list instead of a map?

Reply:
@AkarshSatija  I don't see much difference between list or map for this. This is used for configuring the custom notification messsage